testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -21,62 +21,65 @@
-
+from MDAnalysisTests.datafiles import PDB, XTC, RANDOM_WALK
 
 class TestPCA(object):
     def setUp(self):
         self.u = MDAnalysis.Universe(PDB, XTC)
         self.pca = pca.PCA(self.u.atoms, select='backbone and name CA',
-                           align=True)
+                           align=False)
         self.pca.run()
         self.n_atoms = self.u.select_atoms('backbone and name CA').n_atoms
 
     def test_cov(self):
         atoms = self.u.select_atoms('backbone and name CA')
-        reference = atoms.positions
-
-        ref_cog = atoms.center_of_geometry()
-        reference -= ref_cog
         xyz = np.zeros((10, 642))
         for i, ts in enumerate(self.u.trajectory):
-            mobile_cog = atoms.center_of_geometry()
-            mobile_atoms, old_rmsd = _fit_to(atoms.positions,
-                                             reference,
-                                             mobile_atoms=atoms,
-                                             mobile_com=mobile_cog,
-                                             ref_com=ref_cog)
-            xyz[i] = mobile_atoms.positions.ravel()
+            xyz[i] = atoms.positions.ravel()
 
         cov = np.cov(xyz, rowvar=0)
         assert_array_almost_equal(self.pca.cov, cov, 4)
 
     def test_cum_var(self):
         assert_almost_equal(self.pca.cumulated_variance[-1], 1)
-        # makes no sense to test values here, no physical meaning
 
     def test_pcs(self):
-        assert_equal(self.pca.p_components.shape, (self.n_atoms*3, self.n_atoms*3))
+        assert_equal(self.pca.p_components.shape,
+                     (self.n_atoms*3, self.n_atoms*3))
 
     def test_different_steps(self):
-        self.pca.run()
-        cum_var = self.pca.cumulated_variance
-        pcs = self.pca.p_components
+        dot = self.pca.transform(self.u.select_atoms('backbone and name CA'),
+                                 start=5, stop=7, step=1)
+        assert_equal(dot.shape, (2, self.n_atoms*3))
 
     def test_transform(self):
         self.ag = self.u.select_atoms('backbone and name CA')
         self.pca_space = self.pca.transform(self.ag, n_components=1)
         assert_equal(self.pca_space.shape,
                      (self.u.trajectory.n_frames, 1))
 
-
-    # Accepts universe as input, but shapes are not aligned
+    # Accepts universe as input, but shapes are not aligned due to n_atoms
     @raises(ValueError)
     def test_transform_inputs(self):
         self.ag = self.u.select_atoms('backbone and name CA')
         self.pca_space = self.pca.transform(self.u, n_components=1)
         assert_equal(self.pca_space.shape,
                      (self.u.trajectory.n_frames, 1))
+
+
+    def test_cosine_content(self):
+        rand = MDAnalysis.Universe(RANDOM_WALK)
+        pca_random = pca.PCA(rand.atoms).run()
+        dot = pca_random.transform(rand.atoms)
+        content = cosine_content(dot, 0)
+        assert_almost_equal(content, .99, 1)
+
+
+def cosine_content(pc, i):
```

Comment:
this could also live in the PCA module with a reference to the papers by Berk Hess.